### ISSUE TITLE
Remove Html module and make Html2 the default

### ISSUE THREAD

**Issue by pbiggar** [pbiggar/rescript-tea#13]
The flat name space is less idiomatic than the nested modules in HTML2

**Comment by OceanOak** [comment]
Html module is only used inside test files and tea_html2.res

This is where it is used in tea_html2.res. Should I move them from tea_html.res to tea_html2.res

and remove tea_html.res file?

```
let on = Tea_html.on

  let onWithOptions = Tea_html.onWithOptions

  let defaultOptions = Tea_html.defaultOptions

  let targetValue = Tea_html.targetValue

  let targetChecked = Tea_html.targetChecked

  let keyCode = Tea_html.keyCode
```

**Comment by pbiggar** [comment]
Yes, exactly!

**Comment by pbiggar** [comment]
Fixed in #17
